homeassistant/components/media_player/plex.py

```diff
@@ -154,11 +154,14 @@ def update_devices():
             return
 
         new_plex_clients = []
+        available_ids = []
```

Comment:
I would prefer `available_client_ids` here.

Reply:
Yes that would make more sense